ansible/roles/apache/tasks/Solaris.yml

```diff
@@ -0,0 +1,37 @@
+# Works on Solaris 11
+# - name: Install solaris wazuh agent custom package
+#   become: true
+#   shell: pkg install web/server/{{ program }}
+#   environment:
+#     PATH: "/opt/python3/bin/:/usr/sbin:/usr/bin:/usr/sbin/:/opt/csw/gnu/:/usr/sfw/bin/:/opt/csw/bin/"
+
+- name: Install CSWpkgutil
+  become: true
+  shell: pkgadd -d http://get.opencsw.org/now -n
+  environment:
+    PATH: "/opt/python3/bin/:/usr/sbin:/usr/bin:/usr/sbin/:/opt/csw/gnu/:/usr/sfw/bin/:/opt/csw/bin/"
```

Comment:
Repeated PATH variable, prone to errors.

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/3032/commits/60ed4a54e770f5948dee61bc5b10e51173d843d1